web_src/css/repo.css

```diff
@@ -143,41 +143,37 @@
   margin-bottom: 12px;
 }
 
-.repository #clone-panel #repo-clone-url {
+.repository .clone-panel #repo-clone-url {
   width: 320px;
   border-radius: 0;
 }
 
 @media (min-width: 768px) and (max-width: 991.98px) {
-  .repository #clone-panel #repo-clone-url {
+  .repository .clone-panel #repo-clone-url {
     width: 200px;
   }
 }
 
 @media (max-width: 767.98px) {
-  .repository #clone-panel #repo-clone-url {
+  .repository .clone-panel #repo-clone-url {
     width: 200px;
   }
 }
 
-.repository #clone-panel #repo-clone-https,
-.repository #clone-panel #repo-clone-ssh {
-  border-right: none;
-}
-
-.repository #clone-panel #more-btn {
-  border-left: none;
+.repository .ui.action.input.clone-panel > button + button,
+.repository .ui.action.input.clone-panel > button + input {
+  margin-left: -1px; /* make the borders overlap to avoid double borders */
```

Comment:
Couldn't we instead remove the double border?

Reply:
Nope, it would be quite hacky to "remove" the border. I didn't find a clearer method other than this, I think the current margin approach is good enough and we have used it a lot in code base.